test/test_torch.py

```diff
@@ -3851,6 +3851,8 @@ def test_serialization(self):
         for use_name in (False, True):
             with tempfile.NamedTemporaryFile() as f:
                 handle = f if not use_name else f.name
+                if sys.platform == 'win32' and use_name:
+                    handle = tempfile.mktemp()
```

Comment:
I'm confused now. We have already created a temporary file, so why do this?

Reply:
Not supported in win32.